src/VisualQueryBuilder/types.ts

```diff
@@ -0,0 +1,121 @@
+/**
+ * Shared types that can be reused by Loki and other data sources
```

Comment:
Not sure if this comment is really necessary.

Reply:
fixed in https://github.com/grafana/grafana-experimental/pull/98/commits/a594bdbf7277670de3fe92f74d8da86212959bc9